### PR TITLE
Move Leadership, General and Job Listing web content, and structures to Asset Libraries

### PR DESCRIPTION
**Pull request by mirandastephane**

### CODE REVIEW

**Comment by mirandastephane** [comment]
@egnajek, would you be able to review this, please

**Review by egnajek — CHANGES_REQUESTED** [review]
@mirandastephane - Thank you for working on these tasks. Please see below, and let me know if you have any questions!

- Please rename 'Job Listing' Asset Library to 'HR'
- Please create a folder inside the asset library called 'HR' and move the Job Listings themselves into the folder
- Please rename the label 'Key Responsabilities' to 'Key Responsibilities' in the Job Listing structure
- There are 10 job listings in clarityvisionsolutions.com website and 9 job listings in the local environment. It looks like Warehouse Manager is the one missing. Are we creating this one during the exercises? If not, let's add it for consistency between final product and the bootcamp workspace.
- Please rename 'Leadership Headshots' Asset Library to just be 'Leadership'
- Please add paragraph breaks to Leadership article for Diego Morales (please see clarityvisionsolutions.com for reference)
- Please move the General articles from the 'General' Asset Library to the 'Marketing Assets' Asset Library
- Please delete the 'General' Asset Library

File: configs/local/data/license/LiferayActivationKey_DXPDevelopment_developer.li [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+rO0ABXc0AAAABAAWTGlmZXJheSBBY3RpdmF0aW9uIEtleQAWTGlmZXJheSBBY3RpdmF0aW9uIEtleXNyAA5qYXZhLnV0aWwuRGF0ZWhqgQFLWXQZAwAAeHB3CAAAAZQclk4AeHVyABNbTGphdmEubGFuZy5TdHJpbmc7rdJW5+kde0cCAAB4cAAAAAB1cQB+AAIAAAAAegAAAS8BCDZjNTI0ZGU5ZWZjYjU0YWYwMjlkODVjMGQwYzQzOWM1NDk4NGYzYzI0NzY1Njk3ZWMwZDMxODQ1ZjVlY2EwYWQzMzA0ZjBjZGE1ZWZmYTNmMzYwYThlNDcwMDNkNmVmOGU2NGNkNDYxZDdhMzFhOTNkNjAzYjNmNTAyZDk2MmVmNmM4MDkxZDhiNjYxNDc0MjY1OTllZDk1ODdmZTE4ZDlkZGZlN2JiZjM3NmY3NjBlOTgzOWMyN2I1NDIzMjlkMWU2ZDk0NjE5MGM5MWJiYjViM2IyZmYyZGU3YWZhMTYwYzIwYThlZmJlYmM1OTAxNGVjN2ExNzA2MmI0NDRlYzY4NjEzYjVmNAAAAZFRwEvLAA1EWFAgRGV2ZWxvcGVyAAlkZXZlbG9wZXIAATV1cQB+AAIAAAAAd1AAAAAKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWTGlmZXJheSBBY3RpdmF0aW9uIEtleQAPRFhQIERldmVsb3BtZW50AAZQb3J0YWwAAzcuNHVxAH4AAgAAAABzcQB+AAB3CAAAAYzE5SIAeA==
```
Comment (egnajek) [inline configs/local/data/license/LiferayActivationKey_DXPDevelopment_developer.li]: Is this change required?

Comment (egnajek) [reply configs/local/data/license/LiferayActivationKey_DXPDevelopment_developer.li]: Or the serverId one?

**Review by egnajek — CHANGES_REQUESTED** [review]

File: configs/local/data/license/LiferayActivationKey_DXPDevelopment_developer.li (hunk 1 above)
Comment (mirandastephane) [reply configs/local/data/license/LiferayActivationKey_DXPDevelopment_developer.li]: Unsolicited. I reverted to the previous version.

**Comment by mirandastephane** [comment]
@egnajek, tweaks applied:

- Please rename 'Job Listing' Asset Library to 'HR'  ✅ 
- Please create a folder inside the asset library called 'HR' and move the Job Listings themselves into the folder ✅ 
- Please rename the label 'Key Responsabilities' to 'Key Responsibilities' in the Job Listing structure ✅ 
- There are 10 job listings in clarityvisionsolutions.com website and 9 job listings in the local environment. It looks like ✅ Warehouse Manager is the one missing. Are we creating this one during the exercises? If not, let's add it for consistency between final product and the bootcamp workspace.
- Please rename 'Leadership Headshots' Asset Library to just be 'Leadership' ✅ 
- Please add paragraph breaks to Leadership article for Diego Morales (please see clarityvisionsolutions.com for reference) ✅ 
- Please move the General articles from the 'General' Asset Library to the 'Marketing Assets' Asset Library ✅ 
- Please delete the 'General' Asset Library ✅

**Comment by egnajek** [comment]
Thanks @mirandastephane for making these changes. I do see an error in the Careers and About Us pages. Could this be due to moving the General article web content?
image

**Comment by mirandastephane** [comment]
@egnajek, issues resolved. Please check again. ;)

**Review by egnajek — APPROVED** [review]
Thanks @mirandastephane!
